boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20-memory_map.dtsi

```diff
@@ -172,29 +163,27 @@
 		};
 
 		shared_ram3x_region: memory@2fc12000 {
```

Comment:
I don't have a strong reason to keep it. There's a similar situation with the `shared_ram20_region` node, which also hasn't been `nordic,owned-memory` compatible for a while, so it could probably be flattened as well. The only thing is that I would want to rename `cpuapp_dma_region` to `cpuapp_dma_ram3x_region` etc. to be more consistent; this would create a bigger diff, since these nodes are referenced in several overlays.

Reply:
I agree with your proposal. I believe the additional nesting was caused by hardware limitations that were recently removed. So we can simplify.

Reply:
Updated. I changed my mind about the label rename, particularly because I wanted to keep `ipc_shm` labels as is.